gazebo/rendering/CustomPSSMShadowCameraSetup.cc

```diff
@@ -464,6 +464,16 @@ void CustomPSSMShadowCameraSetup::calculateShadowMappingMatrix(
   }
 }
 
+//////////////////////////////////////////////////
+Ogre::Matrix4 CustomPSSMShadowCameraSetup::buildSimplePerspectiveMatrix(
```

Comment:
Since this function is only used within this translation unit, and it does not depend on member variables, I suggest only declaring it in this .cc file as a static free function, e.g.

```cpp
static Ogre::Matrix4 buildSimplePerspectiveMatrix(const Ogre::Real _near, const Ogre::Real _far)
{
  ...
```

And then we don't need to modify the header file.

Reply:
I can do that. Out of curiosity, does adding this method break and project rules? I don't believe it will break ABI.

Reply:
> I can do that. Out of curiosity, does adding this as a method break and project rules? I don't believe it will break ABI.

since it is not a `virtual` function, it would not break ABI to add it to the installed header file, but keeping it in the `.cc` file helps limit our API surface area

Reply:
Done